policyengine_us/parameters/gov/states/ar/tax/income/rates/main/rate.yaml

```diff
@@ -24,7 +24,7 @@ metadata:
     href: https://www.dfa.arkansas.gov/wp-content/uploads/TaxBrackets_2023.pdf#page=1
   - title: 2024 Indexed Tax Brackets
     href: https://www.dfa.arkansas.gov/wp-content/uploads/2024_TaxBrackets.pdf#page=1
-  - title: Bill to reduce the 2024 income tax HB1001 
+  - title: Bill signed into law to reduce the 2024 income tax HB1001
```

Comment:
Let's remove this reference as it's now reflected in statutes: https://law.justia.com/codes/arkansas/title-26/subtitle-5/chapter-51/subchapter-2/section-26-51-201/